src/vin2dot.c

```diff
@@ -25,8 +27,137 @@ void yyerror(char *s, ...) {
 	va_end(ap);
 }
 
-int main() {
+enum command {
+	CMD_AST,
+	CMD_AST_AFTER,
+	CMD_AST_AND_SCOPES,
+	CMD_SCOPES,
+};
+
+static const char* pop_argv(int *argc, char **argv[]) {
+	if (argc == 0)
+		return NULL;
+
+	const char *ret = **argv;
+
+	(*argc)--;
+	(*argv)++;
+
+	return ret;
+}
+
+static void show_usage(const char *prgname) {
+	fprintf(stderr, "usage: %s <COMMAND>\n\n", prgname);
+	fprintf(stderr, "Commands:\n");
+	fprintf(stderr, "  ast\tShow the AST in the dot format\n");
```

Comment:
```suggestion
	fprintf(stderr, "  %-10s\t%s\n", "ast", "Show the AST in the dot format");
```

Reply:
Maybe those should be left for when we make a better error system

Reply:
ok